### ISSUE TITLE
status bar to display shape properties

### ISSUE THREAD

**Issue by njourdane** [bernhard-42/vscode-ocp-cad-viewer#122]
I find the option "Show shape properties" very useful.

A drawback is that the box takes a lot of place and we have to click on the shape to display the properties.

I think it could be very practical to turn this into a status bar that updates when the cursor hovers a shape, among with an option to display/hide it by default.

Here is a mockup:

![image](https://github.com/user-attachments/assets/445cbb0c-6fa1-49eb-9008-a080c92b8280)

**Comment by bernhard-42** [comment]
Agreed, this is another way to show the info, but I currently do not plan to change the behavior of the viewer. Some other topics have higher priority

**Comment by bernhard-42** [comment]
I looked more into it and will not change the current behaviour. 
Retrieving the info sometimes needs to long (extension => python => BRep commands => Python => extension roundtrip) which leads to a non-optimal experience
